src/components/Footer.js

```diff
@@ -181,22 +176,35 @@ const Footer = ({ isOpen, toggleFooter }) => {
                 </CanaryContent>
               </FooterDivCanary>
               <FooterDivLinks>
-                <ImageAndTextLink
+                <OutboundLink
                   target="_blank"
                   rel="noopener noreferrer"
                   href="https://devcon.org"
                 >
-                  <img src={devconLogo} alt="Devcon Logo" />
                   Devcon.org
```

Comment:
This should just be `Devcon`, not `Devcon.org`.

Reply:
fixed